_data/simple-icons.json

```
@@ -7074,6 +7074,12 @@
             "hex": "FF6801",
             "source": "https://www.joinhoney.com"
         },
+        {
+            "title": "Honeybadger",
+            "hex": "EA5A37",
+            "source": "https://www.honeybadger.io",
+            "guidelines": "https://www.honeybadger.io/assets"
```

```suggestion
            "source": "https://www.honeybadger.io/assets",
            "guidelines": "https://www.honeybadger.io/assets"
```

Might as well use the SVGs available on the `assets` page too :)